saleor/plugins/webhook/plugin.py

```diff
@@ -2995,30 +2995,89 @@ def process_payment(
             **kwargs,
         )
 
-    def get_taxes_for_checkout(
-        self, checkout_info, lines, previous_value
-    ) -> Optional["TaxData"]:
-        return trigger_all_webhooks_sync(
-            WebhookEventSyncType.CHECKOUT_CALCULATE_TAXES,
-            lambda: generate_checkout_payload_for_tax_calculation(
-                checkout_info,
-                lines,
-            ),
-            parse_tax_data,
-            checkout_info.checkout,
+    def __run_tax_webhook(
+        self,
+        event_type: str,
+        app_identifier: str,
+        payload: str,
+        subscriptable_object=None,
+    ):
+        app = (
+            App.objects.filter(
+                identifier=app_identifier,
+                is_active=True,
+            )
+            .order_by("-created_at")
+            .first()
+        )
```

Comment:
we could possibly prefetch webhooks since we call `app.webhooks.all()` below

Reply:
I will stick with no prefetch, as I believe it's more clear in code and consistent with other functions in file.

Reply:
Actually changed in fixes: switched to app id